deltachat-ffi/deltachat.h

```diff
@@ -4518,6 +4518,24 @@ int             dc_msg_get_info_type          (const dc_msg_t* msg);
 #define         DC_INFO_INVALID_UNENCRYPTED_MAIL  13
 #define         DC_INFO_WEBXDC_INFO_MESSAGE       32
 
+
+/**
+ * Get link attached to an webxdc info message.
+ * The info message needs to be of type DC_INFO_WEBXDC_INFO_MESSAGE.
```

Comment:
so this will not work for notifications using `update.notify` setting a summary but no info message :thinking: when showing such a notification (which potentially will have info-message sometimes) I guess we will add an "open" to the notification to jump to the app, for notifications with info-messages it would work for not for summary, it is a limitation but probably not a big deal for the first iterations and could be improved at a later point I guess

Reply:
indeed, that is not possible currently. but that is no conceptual flaw, we could probably easily add the hash to the event as well, if we consider that an important usecase as well (or just to make documentation easier :)

however, i suggest to do that after this PR and after all UI have adapted to href approach - that way we can see if things work out as we exepect. if so, adpating UI for event-href is easy :)